src/utils/socket_utils.hpp

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2020, The OpenThread Authors.
```

Comment:
```suggestion
 *  Copyright (c) 2021, The OpenThread Authors.
```

Reply:
It was 2020 because this PR was actually created quite a time ago.